tests/php/ContentReviewReportTest.php

```diff
@@ -52,13 +52,17 @@ public function testPagesDueForReviewReport()
             "ReviewDateBefore" => "2010-12-12",
         ]);
 
-        $this->assertEquals([
-            "Contact Us Child",
-            "Home",
-            "About Us",
-            "Staff",
-            "Contact Us",
-        ], $results->column("Title"));
+        $this->assertListContains([
```

Comment:
wouldn't `testAssertListEquals` be better than contains?

Reply:
Yes. I somehow missed that was a thing. I'll remember for next time 😆